api/src/pcapi/routes/pro/offers.py

```diff
@@ -286,6 +286,8 @@ def patch_publish_offer(
             offer = offers_repository.get_offer_and_extradata(body.id)
             if offer is None:
                 raise api_errors.ApiErrors({"offer": ["Cette offre n’existe pas"]}, status_code=404)
+            if not offers_repository.offer_has_stocks(offer.id):
+                raise api_errors.ApiErrors({"offer": "offer_has_no_stocks"}, 400)
```

Comment:
Un détail en passant par là : quand il n'y en a pas, il y en a encore moins plusieurs :)
```suggestion
                raise api_errors.ApiErrors({"offer": "offer_has_no_stock"}, 400)
```